localstack/services/cloudwatch/provider.py

```diff
@@ -142,6 +143,12 @@ def put_metric_alarm(
     )
 
 
+def convert_camel_to_snake_upper_case(input: str) -> str:
+    # currently we only need to handle "SampleCount" camel case
+    input = re.sub("(.)([A-Z][a-z]+)", r"\1_\2", input)
+    return re.sub("([a-z0-9])([A-Z])", r"\1_\2", input).upper()
```

Comment:
This seems generic enough to be in utils, where there are already functions with very similar purpose:
`localstack.utils.strings.snake_to_camel_case`.

Reply:
ah, yes - thanks for the hint! there is already a `camel_to_snake_case` utility 🙂